docs/sources/mimir/manage/run-production-environment/scaling-out.md

```diff
@@ -78,73 +78,28 @@ You might experience the following challenges when you scale down ingesters:
 
-#### Scaling down ingesters deployed in a single zone (default)
+#### Scaling down ingesters
 
 Complete the following steps to scale down ingesters deployed in a single zone.
 
-1. Configure the Grafana Mimir cluster to discover and query new uploaded blocks as quickly as possible.
+1. Set each ingester to read-only mode:
 
-   a. Configure queriers and rulers to always query the long-term storage and to disable ingesters [shuffle sharding]({{< relref "../../configure/configure-shuffle-sharding" >}}) on the read path:
+   a. Send a POST request to the `/ingester/prepare-instance-ring-downscale` API endpoint on the ingester to place it into read-only mode.
 
-   ```
-   -querier.query-store-after=0s
-   -querier.shuffle-sharding-ingesters-enabled=false
-   ```
+1. Wait until the blocks uploaded by read-only ingesters are available for querying before proceeding. The required amount of time to wait depends on your configuration and is the maximum value for the following settings:
 
-   b. Configure the compactors to frequently update the bucket index:
-
-   ```
-   -compactor.cleanup-interval=5m
-   ```
-
-   c. Configure the store-gateways to frequently refresh the bucket index and to immediately load all blocks:
-
-   ```
-   -blocks-storage.bucket-store.sync-interval=5m
-   -blocks-storage.bucket-store.ignore-blocks-within=0s
-   ```
-
-   d. Configure queriers, rulers and store-gateways with reduced TTLs for the metadata cache:
-
-   ```
-   -blocks-storage.bucket-store.metadata-cache.bucket-index-content-ttl=1m
-   -blocks-storage.bucket-store.metadata-cache.tenants-list-ttl=1m
-   -blocks-storage.bucket-store.metadata-cache.tenant-blocks-list-ttl=1m
-   -blocks-storage.bucket-store.metadata-cache.metafile-doesnt-exist-ttl=1m
-   ```
+- The configured `-querier.query-store-after`
+- Two times the configured `-blocks-storage.bucket-store.sync-interval`
+- Two times the configured `-compactor.cleanup-interval`
 
-1. Scale down one ingester at a time:
+1. Scale down each ingester:
 
-   a. Send a POST request to the `/ingester/shutdown` API endpoint on the ingester to terminate.
+   a. Send a POST request to the `/ingester/shutdown` API endpoint on the ingester to terminate it.
 
    b. Wait until the API endpoint call has successfully returned and the ingester logged "finished flushing and shipping TSDB blocks".
 
    c. Send a `SIGINT` or `SIGTERM` signal to the process of the ingester to terminate.
 
-   d. Wait 10 minutes before proceeding with the next ingester. The temporarily applied configuration guarantees newly uploaded blocks are available for querying within 10 minutes.
-
-1. Wait until the originally configured `-querier.query-store-after` period of time has elapsed since when all ingesters have been shutdown.
-1. Revert the temporary configuration changes done at the beginning of the scale down procedure.
-
-#### Scaling down ingesters deployed in multiple zones
```

Comment:
This section is removed, but the documentation still says "Complete the following steps to scale down ingesters deployed in a single zone", and it's not quite clear how it translates to multiple zones. Procedure for multi-zone setup is the same, but we should mention that and perhaps remove that "single zone" sentence. WDYT?

Reply:
I agree. I reworded that sentence a bit. Lmk what you think.